CMakeLists.txt

```diff
@@ -24,6 +28,7 @@ if (NOT TARGET _pico_sdk_inclusion_marker)
 
     # allow customization
     add_sub_list_dirs(PICO_SDK_PRE_LIST_DIRS)
+    add_sub_list_dirs(PICO_SDK_PRE_LIST_FILES)
```

Comment:
Was this supposed to be `add_sub_list_files` as you've done on line 40? :shrug:

Reply:
yes; oops